build_container.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -e
+
+source .env
+
+docker build -t elife/sciencebeam --build-arg sciencebeam_gym_commit=${SCIENCEBEAM_GYM_COMMIT} .
```

Comment:
this is almost reimplementing `docker-compose build`, but it's fine for now

Reply:
The tag used across building and running this should be `elifesciences/sciencebeam:${commitOfThisProject}` (Notice the organization name). In a Jenkinsfile it can be read with `commit = elifeGitRevision()` e.g. https://github.com/elifesciences/profiles/blob/develop/Jenkinsfile#L6

It's not a big problem for now because we don't push that image anywhere (do we?)

Reply:
I certainly 'borrowed' the _.env_ file from the docker compose setup.  I wasn't sure whether _docker-compose_ was appropriate for a single container.

Reply:
But it's the commit of _sciencebeam-gym_ (the dependency), not this project, _sciencebeam_. No, we don't at present push it anywhere. Although I did realise that there is some duplication in _sciencebeam-texture_ which probably could be avoided by using a pushed image.

Reply:
> But it's the commit of sciencebeam-gym (the dependency)

Yes, they are two unrelated commits:
- the tag of this image should be related to the commit of this project
- the build argument should use the commit of the dependency
A story for another time

Reply:
Okay, apologies, I misread. You were of course referring to the tag of the docker image. We can add that as well (probably as a separate PR).

Reply:
Yes; unrelated, but the prefix should be `elifesciences/`